e2e.Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG IMG_TAG=latest
+
+# Compile the gaiad binary
+FROM golang:1.17-alpine AS gaiad-builder
+WORKDIR /src/app/
+COPY go.mod go.sum* ./
+RUN go mod download
+COPY . .
+ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
+RUN apk add --no-cache $PACKAGES
+RUN CGO_ENABLED=0 make install
+
+# Add to a distroless container
+FROM gcr.io/distroless/cc:$IMG_TAG
```

Comment:
We can use scratch actually, but then we have to manually add certificates and all like this

```dockerfile

FROM alpine as certs
RUN apk add ca-certificates --no-cache
...
FROM scratch
COPY --from=certs /etc/ssl/certs/ /etc/ssl/certs/
...
```

So I would suggest to use `gcr.io/distroless/static` since we are `CGO_ENABLED=0`